packages/framework/ai-collab/src/explicit-strategy/utils.ts

```diff
@@ -58,3 +58,72 @@ export function getOrCreate<K, V>(
 	}
 	return value;
 }
+
+/**
+ * Computes the Levenshtein distance between two strings.
+ * The Levenshtein distance between two strings is the minimum number of single-character edits (insertions, deletions, or substitutions) required to change one string into the other.
+ */
+export function levenshteinDistance(a: string, b: string): number {
```

Comment:
Do we need to implement this ourselves? Is there an external library we could use for this?